tests/runner.js

```diff
@@ -57,10 +58,9 @@ function checkOnlyInTests() {
 function runMocha() {
   console.time('Mocha Tests Running Time');
   mocha.run(function(failures) {
-    process.on('exit', function() {
-      console.timeEnd('Mocha Tests Running Time');
-      process.exit(failures);
-    });
+    console.timeEnd('Mocha Tests Running Time');
+    exit.releaseExit();
```

Comment:
@stefanpenner - This is a temp work around only. Our CI was in a passing state when it should have been failing for ~ 2 weeks. We needed to ensure we could trust it again and begin righting the ship. Once the PR fixing the discovered issues with capture-exit are landed and released, we will PR to remove this.